v2/googlecloud-to-googlecloud/README_Bigtable_Change_Streams_to_BigQuery.md

```diff
@@ -25,12 +25,12 @@ on [Metadata Annotations](https://github.com/GoogleCloudPlatform/DataflowTemplat
 ### Optional parameters
 
 * **writeRowkeyAsBytes** : Whether to write rowkeys as BigQuery `BYTES`. When set to `true`, row keys are written to the `BYTES` column. Otherwise, rowkeys are written to the `STRING` column. Defaults to `false`.
-* **writeValuesAsBytes** : When set true values are written to BYTES column, otherwise to STRING column. Defaults to false.
-* **writeNumericTimestamps** : Whether to write the Bigtable timestamp as BigQuery `INT64`. When set to true, values are written to the `INT64` column. Otherwise, values are written to the `TIMESTAMP` column. Columns affected: `timestamp`, `timestamp_from`, and `timestamp_to`. Defaults to `false`. When set to `true`, the time is measured in microseconds since the Unix epoch (January 1, 1970 at UTC).
+* **writeValuesAsBytes** : When set to `true`, values are written to the BYTES column. Otherwise, values are written to the STRING column. Defaults to: `false`.
```

Comment:
BYTES column and STRING column is about types, not a specific column.
Maybe reword to something like "values are written to a column of type BYTES, otherwise to a column of type STRING"

Reply:
Changed in BigtableChangeStreamToBigQueryOptions.java